suites/nautilus/ansible/tier_2_deploy_rpm_ceph.yaml

```diff
@@ -0,0 +1,220 @@
+#############################################################################
+# Automation support for RHCS4 cluster deployment with lvm osd scenario
+# Cluster configuration: (conf/nautilus/ansible/tier_2_deploy.yaml)
+# --------------------------------------------------------------------------
+# Nodes:
+#   - 3 MONS, 1 MGRs, 3 OSDs, 1 RGW, 1 GRAFANA, 1 CLIENT
+#
+# Test steps:
+# ---------------------------------------------------------------------
+# - Deploy RHCS4 Nautilus Ceph cluster using CDN RPMS with lvm osd scenario
+# - Check idempotency of site.yaml playbook
+# - Run some I/O's
+# - Add mon , osd, nfs, client, rgw, mds using rollover
+# - Shrink mon and osd
+#
+#############################################################################
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-83571467
+      module: test_ansible.py
+      config:
+        is_mixed_lvm_configs: True
+        ansi_config:
+            ceph_test: True
+            ceph_origin: distro
+            ceph_stable_release: nautilus
+            ceph_repository: rhcs
+            osd_scenario: lvm
+            journal_size: 1024
+            ceph_stable: True
+            ceph_stable_rh_storage: True
+            fetch_directory: ~/fetch
+            copy_admin_key: true
+            dashboard_enabled: True
+            dashboard_admin_user: admin
+            dashboard_admin_password: p@ssw0rd
+            grafana_admin_user: admin
+            grafana_admin_password: p@ssw0rd
+            node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+            grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+            prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+            alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+            ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_default_pool_size: 2
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
+                client:
+                  rgw crypt require ssl: false
+                  rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+                    testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+            cephfs_pools:
+              - name: "cephfs_data"
+                pgs: "8"
+              - name: "cephfs_metadata"
+                pgs: "8"
```

Comment:
This setting would result in the `ceph cluster health` to be in a warn state. I believe it should be `16`